packages/react-dom-bindings/src/events/plugins/ChangeEventPlugin.js

```diff
@@ -335,8 +336,12 @@ function extractEvents(
   }
 
   // When blurring, set the value attribute for number inputs
-  if (domEventName === 'focusout') {
-    handleControlledInputBlur(((targetNode: any): HTMLInputElement));
+  if (domEventName === 'focusout' && targetInst) {
+    // These props aren't necessarily the most current but we warn for changing
+    // between controlled and uncontrolled, so it doesn't matter and the previous
+    // code was also broken for changes.
+    const props = targetInst.memoizedProps;
```

Comment:
nit: pass isControlled (props.value != null) here instead of having it in the callee? since the props can be stale, seems better to localize the calculations to the same place where we know they are stale